src/App.js

```diff
@@ -1,11 +1,13 @@
-import {Route, Switch, Redirect} from 'react-router-dom'
+import { Route, Switch, Redirect } from 'react-router-dom'
 import './App.css'
 import Header from './components/common/Header'
 import Home from './components/Home'
+import AllProductsSection from './components/products'
 
 const App = () => (
   <Switch>
```

Comment:
Instead of the React Router V5 use the latest react router version.

References:
https://reactrouter.com/en/main
https://blog.logrocket.com/react-router-v6-guide/